api/controllers/case.js

```diff
@@ -93,16 +98,15 @@ router.post('/new', function (req, res, next) {
     res.status(401).json({message: 'access denied - user does not have proper authorization'})
   }, function () {
     es.index({
-      index: 'pp',		
-      type: 'case',		
-      body: req.body		
-    }, function (error, response) {		
-      if (error) {		
-        res.status(error.status).json({message: error.message})		
-      } else {		
-        // console.log(response)		
-        res.status(200).json(req.body)		
-      }		
+      index: 'pp',
```

Comment:
Is the idea to do another pass w/ new API conversions later?  This will index in elasticsearch, but not in SQL, so any tests of this (not that we have any) would not work.